wic/src/components/FoodLocations.js

```diff
@@ -1,48 +1,43 @@
 import React, {PropTypes, Component} from 'react';
 import ReactDOM from 'react-dom';
-
-import shouldPureComponentUpdate from 'react';
-
-import Gmap from './Gmap';
-
+// import shouldPureComponentUpdate from 'react';
```

Comment:
I'd prefer if you deleted, instead of just left the commented out line.

Reply:
removed.